rpcs3/Emu/RSX/VK/VKGSRender.h

```diff
@@ -280,15 +284,15 @@ struct flush_request_task
 	{
 		while (num_waiters.load() != 0)
 		{
-			_mm_pause();
+			num_waiters.wait(num_waiters);
```

Comment:
Yep, a pitfall. Which value was read from load(), the same should go to wait() argument, otherwise it may freeze.

Reply:
Oh, thanks!